projects/github.com/cosmtrek/air/package.yml

```diff
@@ -0,0 +1,74 @@
+# this is a basic template for packages we strongly recommend looking at the
+# contents of both:
+#
+# * https://github.com/pkgxdev/pantry
+#
+# as a learning guide and reference. For example, if you are package a `go`
+# project, then browse the listings and find a go package.
+
+
+# this is the source tarball. Swap the version out for {{version}} or we won’t
+# be able to upgrade the package when it updates!
+distributable:
+  url: https://github.com/cosmtrek/air/archive/v{{version}}.tar.gz
```

Comment:
should be:

https://github.com/cosmtrek/air/archive/refs/tags/v{{version}}.tar.gz

it looks like